### PR TITLE
Improvement/adds sellers

### PR DESCRIPTION
**Pull request by lucasayb**
#### What is the purpose of this pull request?
Adds `sellers` attribute for the orderForm query

#### What problem is this solving?
Adds `sellers` attribute for the orderForm query

#### How should this be manually tested?
This app is linked at the workspace https://lucas--codeby.myvtex.com/

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [X] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by ullayne02** [comment]
Just remembering to fill the release and doc bots :smile_cat:

**Comment by lucasayb** [comment]
> Just remembering to fill the release and doc bots 😸

I don't think I have anything to add in the `docs` having in mind that I'm just adding an attribute

**Comment by ullayne02** [comment]
> > Just remembering to fill the release and doc bots smile_cat
> 
> I don't think I have anything to add in the `docs` having in mind that I'm just adding an attribute

Yes! But you can check `There's nothing new to document :thinking:` and for the release bot: Minor. Just to build it

**Comment by lucasecdb** [comment]
@lucasayb can you rebase?

File: react/queries/orderForm.gql [hunk 1]
```diff
@@ -1,6 +1,7 @@
 query orderForm {
   orderForm {
     cacheId
+    sellers
```
Comment (lucasecdb) [inline react/queries/orderForm.gql:4]: this property doesn't exist in the [orderForm schema](https://github.com/vtex-apps/store-graphql/blob/master/graphql/types/OrderForm.graphql#L1-L26).

**Comment by lucasayb** [comment]
> I'm having an issue linking this branch
> image
> 
> That might be because of what @lucasecdb said

Should I open a pull request for Store Graphql then?

**Comment by lucasayb** [comment]
Hey everyone, sorry for the delay. I've opened the pull request that adds the needed `Types` to the OrderForm schema

https://github.com/vtex-apps/store-graphql/pull/459

File: react/queries/orderForm.gql (hunk 1 above)
Comment (lucasecdb) [reply react/queries/orderForm.gql:4]: @lucasayb this field is not a scalar, can you specify which sub-fields do you want to query from it?

**Comment by lucasecdb** [comment]
hey @lucasayb just checking, do you still need the changes in this PR? I replied to my thread ([here](https://github.com/vtex-apps/store-resources/pull/110#discussion_r417569110)) asking about the fields you need from the `sellers` field.

**Comment by lucasayb** [comment]
> hey @lucasayb just checking, do you still need the changes in this PR? I replied to my thread ([here](https://github.com/vtex-apps/store-resources/pull/110#discussion_r417569110)) asking about the fields you need from the `sellers` field.

I've fixed what was requested!

**Pull request merged**